### DISCUSSION TITLE
A little bit of confusion about the definition of "potential function"

### DISCUSSION THREAD

**Discussion by 343333333** [deepmodeling/deepmd-kit#1320] [Q&A]
For the model trained by DPMD, whether it is the neural network potential of element type or the system of ALL the atom. For example,  if I train with 64 oxygens, what I get is a potential of 128 oxygens, or a potential of one oxygen.
Or more realistically, can I simulate the dynamics of 128 or more oxygen with the potential of 64 oxygen?
thx

**Comment by njzjz** [comment]
Yes, the size is variable, as the model is applied to each atom.
